src/components/organize/TagsManager/TagChip.tsx

```diff
@@ -27,6 +30,12 @@ const useStyles = makeStyles<Theme, StyleProps>(() => ({
     cursor: ({ clickable, disabled }) =>
       clickable && !disabled ? 'pointer' : 'default',
     display: 'flex',
+    fontSize: ({ size }) =>
+      ({
+        large: '1.2em',
+        medium: '1.0em',
+        small: '0.8em',
+      }[size]),
```

Comment:
:+1:

Reply:
Because you know how much I hate `switch` 😉